docs/.vuepress/config.js

```diff
@@ -71,17 +71,19 @@ const sidebarUserGuide = [
           ['/HardwareWalletIntegration', 'Hardware Wallet Integration']
         ]
       },
-
       ['/Invoices', 'Invoices'],
       ['/PaymentRequests', 'Payment Requests'],
-
       {
         title: 'Pull Payments',
         path: '/PullPayments',
         children: [['/Refund', 'Refunds']]
       },
       ['/Payouts', 'Payouts'],
-      ['/Forms', 'Forms'],
+      {
+        title: 'Form Builder',
+        path: '/forms',
```

Comment:
Let's stick to the camelcase convention here:

```suggestion
        path: '/Forms',
```